test/functional/test_framework/test_node.py

```diff
@@ -375,7 +375,7 @@ def test_success(cmd):
                 stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL) == 0
 
         if not sys.platform.startswith('linux'):
-            self.log.warning("Can't profile with perf; only availabe on Linux platforms")
+            self.log.warning("Can't profile with perf; only available on Linux platforms")
```

Comment:
How is aa94f98fdd296166859913ec74671f572a7e6972 relevant to this PR?

Reply:
Hello @PastaPastaPasta, I have dropped those commits now,

I added aa94f98 change because I saw CI failing once with the below error, I was trying to make CI happy, It was just a warning, but yesterday CI behavior was very flaky
```
^ Warning: codespell identified likely spelling errors. Any false positives? Add them to the list of ignored words in test/lint/lint-spelling.ignore-words.txt
Uploading artifacts for failed job
00:01
Uploading artifacts...
WARNING: /builds/dashpay/dash/build-ci: no matching files 
ERROR: No files to upload         
```
40ba7d5 This change is my error in understanding, I thought correcting obsolete filename would be a good thing to do,  (how should we approach these ?, Shall we open a separate PR or create an issue to track it )

Reply:
Yes, creating a separate PR would be preferable